submariner/fine-grained-connectivity.md

```diff
@@ -0,0 +1,86 @@
+# Fine Grained Connectivity
+
+## Summary
+
+Submariner currently provisions full mesh topology by default between the participating clusters.
+For some use-cases, we need fine-grained connectivity options. This proposal covers only the topology
+aspect of connectivity.
+
+This proposal aims to address the below use-case
+
+Hub-Spoke Topology -  Here only the client can talk to the server but not to the other clients to
+which the server is connected to.
+
+To achieve each hub and spoke will be grouped together into an isolated group of clusters called Clusterset.
+There will be a full mesh topology within the Clusterset, though in this use case it will be just two clusters
+in the Clusterset. A cluster can be a part of multiple Clusterset. So the hub can be a part of multiple
+Clustersets, where it connects with a spoke in each Clusterset
+
+## Proposal
+
+### Terminology
+
+#### Clusterset
+
+A Clusterset is a group of clusters that forms a trust boundary. The design of Clusterset can be aligned
+to the MCS KEP. More details are available in the below link.
```

Comment:
You mention the MCS KEP (https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api) but then link to the Cluster ID KEP?